packages/backend/src/apps/vault-workspace/actions/create-row/index.ts

```diff
@@ -46,7 +58,9 @@ export default defineAction({
 
     const row: { [key: string]: string } = {}
     for (let i = 0; i < columns.length; i++) {
-      row[columns[i]] = values[i]
+      const columnName = decodeURIComponent(columns[i])
```

Comment:
This might overcount - e.g. if the original variable value was `%40,`, we end up with `@,` instead of `%40,`.

Can we just do a direct inverse of our processVariables
```
row[columns[i]] = values[i].replace(/%2C/g, ',').replace(/%22/g, '"')
```

Also thinking no need to decode columns since columns field cannot contain variables.